template/global-dojo.tmpl

```diff
@@ -1,3 +1,41 @@
+{{- /* dojo.grafana.url */ -}}
+{{- /* URL for the grafana instance. */ -}}
+{{- define "dojo.grafana.url" -}}
+    {{- "https://example.grafana.net/" -}}
+{{- end -}}
+
+{{- /* dojo.grafana.alerting.list.data_source */ -}}
+{{- /* Data source for alert list (alerting/list?dataSource=XXX). */ -}}
+{{- define "dojo.grafana.alerting.list.data_source" -}}
+    {{- "DATASOURCE_NAME" -}}
+{{- end -}}
+
+{{- /* dojo.grafana.alerting.alertmanager */ -}}
+{{- /* AlertManager instance (silences?alertmanager=XXX). */ -}}
+{{- define "dojo.grafana.alerting.alertmanager" -}}
+    {{- "ALERTMANAGER_NAME" -}}
+{{- end -}}
+
+{{- /* dojo.emoji(Data) */ -}}
+{{- /* An emoji as a function of firing alerts and the urgency. */ -}}
+{{- define "dojo.emoji" -}}
+    {{- if eq .Status "firing" -}}
+        {{- with .CommonLabels.urgency -}}
+            {{- if eq . "high" -}}
+                {{- "💥" -}}
+            {{- else if eq . "low" -}}
+                {{- "⚠️" -}}
+            {{- else -}}
+                {{- "💩" -}}
```

Comment:
Just my own opinion, but poop emoji doesn't seem helpful / a bit out of place

Reply:
Warning sign is already in use for low urgency alerts, poop is used when there's no urgency defined. I'm open to suggestions which emoji to use (suggestion can be a PR).